docs/main/concepts/basics.md

```diff
@@ -12,9 +12,7 @@ keywords:
 description: This document provides you with a quick overview of the Kubernetes concepts you need to know for running Stateful Workloads.
 ---
 
-:::note
-This page provides you with a quick overview of the [Kubernetes concepts](https://kubernetes.io/docs/concepts/storage/) you need to know for running Stateful Workloads. If you are already familiar with running Stateful workloads in Kubernetes, header over to the next section on [Container Native Storage](container-native-storage.md).
-:::
+This page provides you with a quick overview of the [Kubernetes concepts](https://kubernetes.io/docs/concepts/storage/) you need to know for running Stateful Workloads. If you are already familiar with running Stateful workloads in Kubernetes, head over to the next section on [Container Native Storage](container-native-storage.md).
```

Comment:
The link text could be Kubernetes Storage Concepts

Reply:
This is the link that we're getting when we search for Kubernetes Storage Concepts. :) and this is the official docs for Kubernetes